package.json

```diff
@@ -18,26 +18,35 @@
   },
   "license": "ISC",
   "author": "Bruno Barbieri",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
-    "index.js"
+    "dist/"
   ],
   "scripts": {
-    "lint": "yarn lint:eslint && yarn lint:misc --check",
+    "build": "tsc --project tsconfig.build.json",
+    "build:clean": "rimraf dist && yarn build",
+    "build:docs": "typedoc",
+    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn lint:dependencies",
+    "lint:dependencies": "depcheck",
     "lint:eslint": "eslint . --cache --ext js,ts",
-    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn lint:dependencies",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
-    "test": "mocha"
+    "prepack": "./scripts/prepack.sh",
+    "test": "mocha -r ts-node/register src/**/*.ts"
   },
   "resolutions": {
+    "@trezor/connect@9.0.6": "patch:@trezor/connect@npm%3A9.0.6#./.yarn/patches/@trezor-connect-npm-9.0.6-37abefdc3e.patch",
+    "@types/web": "0.0.69",
     "mocha/mkdirp": "^0.5.1"
   },
   "dependencies": {
     "@ethereumjs/tx": "^4.0.0",
     "@ethereumjs/util": "^8.0.0",
     "@metamask/eth-sig-util": "^5.0.2",
+    "@metamask/utils": "^4.0.0",
     "@trezor/connect-plugin-ethereum": "^9.0.1",
-    "@trezor/connect-web": "^9.0.6",
+    "@trezor/connect-web": "9.0.6",
```

Comment:
Hmm, do we need to pin this? I see that this is the latest version today, but pinning this would obstruct updating this on the clients in the future.

The patch relies on it being this version, but the patch is only needed for building. The types added by the patch aren't included in the build output, they aren't even referenced. So the patch doesn't require us to force users of this package to stick with this version.

Reply:
Good point, I just unpinned it